.github/workflows/docker-bases.yml

```diff
@@ -31,6 +31,18 @@ jobs:
             dockerfile: './docker/Dockerfile.nvidia'
             runner: ["self-hosted", "gpu", "docker", "openmp"]
 
+          - tag: 'devitocodes/bases:nvidia-nvcc'
+            arch: 'arch=nvcc'
+            version: ''
+            dockerfile: './docker/Dockerfile.nvidia'
+            runner: ["self-hosted", "gpu", "docker", "openmp"]
+
+          - tag: 'devitocodes/bases:nvidia-nvc-host'
```

Comment:
I would call this one bases:cpu-nvc since it's the cpu code but no strong opinion about it

Reply:
ahhhh you're very right! will change